test/commands/percy-command.test.ts

```diff
@@ -5,20 +5,24 @@ describe('percy-command', () => {
     .stub(process, 'env', {PERCY_TOKEN: ''})
     .stderr()
     .command(['percy-command'])
-    .catch(err => expect(err.message).to.equal(
-      'You must set PERCY_TOKEN'
+    .do(output => expect(output.stderr).to.contain(
+      'Warning: You must set PERCY_TOKEN'
     ))
-    .it('requires PERCY_TOKEN to be set')
+    .it('warns about PERCY_TOKEN to be set')
 
   test
-    .stub(process, 'env', {PERCY_TOKEN: ''})
+    .stub(process, 'env', {PERCY_ENABLE: '0', PERCY_TOKEN: ''})
```

Comment:
What happens if I pass an empty string with whitespace? `PERCY_TOKEN: '          '`?

Looks like it might not throw this warning? 

```
  percyTokenPresent(): boolean {
    return this.percyToken !== ''
  }
```

Reply:
I think whitespace would be an "incorrect token", whereas a blank string should be correctly treated as falsey.

Reply:
Yeah that's true. A trim there could prevent the round trip though. `this.percyToken.trim()` 

Pretty small detail :p

Reply:
Fixed with ea0318625fdbc3fe58215e8194376529dc1e3089